client/src/components/shared/search/filter.js

```diff
@@ -7,11 +7,11 @@ import ResetButton from '../button/reset';
 
 class Filter extends Component {
   render() {
-    const { info } = this.props;
+    const { info, showDate } = this.props;
     return (
       <ButtonToolbar className="justify-content-center">
         <Context info={info} />
-        <DateMenu />
+        {showDate === true && <DateMenu /> }
```

Comment:
You can write {showDate && <DateMenu /> }
its shortly

Reply:
Thank you. Fixed!